aip/general/0192.md

```diff
@@ -108,6 +108,12 @@ colloquial use that avoiding it would obscure the reference (example: [IBM][]).
 Comments **should** spell and capitalize trademarked names consistent with the
 trademark owner's current branding.
 
+### Deprecations
+
+To deprecate a service or RPC, the `deprecated` option **must** be set to
```

Comment:
Nit,suggestion: link to the protobuf deprecated option on the word "option"

https://developers.google.com/protocol-buffers/docs/proto#options

Reply:
Done.